app/_hub/kong-inc/kafka-upstream/0.1.0.md

```diff
@@ -0,0 +1,258 @@
+---
+name: Kafka Upstream
+publisher: Kong Inc.
+version: 1.3-x
+# internal 0.0.2, only former version 0.0.1.
+
+desc: Transform requests into Kafka messages in a Kafka topic
+description: |
+   This plugin transforms requests into [Kafka](https://kafka.apache.org/) messages
+   in an [Apache Kafka](https://kafka.apache.org/) topic. For more information, see
+   [Kafka topics](https://kafka.apache.org/documentation/#intro_concepts_and_terms).
+
+   Kong also provides a Kafka Log plugin for publishing logs to a Kafka topic.
+   See [Kafka Log](/hub/kong-inc/kafka-log/).
+
+type: plugin
+enterprise: true
+plus: true
+categories:
+  - transformations
+
+kong_version_compatibility:
+    community_edition:
+      compatible:
+    enterprise_edition:
+      compatible:
+        - 2.4.x
+        - 2.3.x
+        - 2.2.x
+        - 2.1.x
+        - 1.5.x
+        - 1.3-x
+
+params:
+
+  name: kafka-upstream
+  dbless_compatible: yes
+  config:
+    - name: bootstrap_servers
+      required: true
+      value_in_examples: <BOOTSTRAP_SERVERS>
+      urlencode_in_examples: true
+      default:
+      datatype: set of record elements
+      description: |
+        Set of bootstrap brokers in a `{host: host, port: port}` list format.
+    - name: topic
+      required: true
+      value_in_examples: <TOPIC>
+      urlencode_in_examples: true
+      default:
+      datatype: string
+      description: |
+         The Kafka topic to publish to.
+    - name: timeout
+      required: false
+      default: "`10000`"
+      value_in_examples: 10000
+      datatype: integer
+      description: |
+         Socket timeout in milliseconds.
+    - name: keepalive
```

Comment:
note to self: this was renamed from `keepalive` -> `keepalive_timeout`.

A new option called `timeout` was added

Reply:
Reverted the `keepalive -> keepalive_timeout` changes here: https://github.com/Kong/kong-plugin-kafka-upstream/pull/11 due to backwards compatibility reasons.

Added a config option `keepalive_enabled`. Docs are updated